### PR TITLE
EIP proposal: Subscription Token Standard for NFTs and Multi Tokens

### PR DESCRIPTION
**Pull request by julesl23**
Please find this new proposal in file:
EIP-Subscription_Token_Standard.md

### CODE REVIEW

File: EIPS/EIP-Subscription_Token_Standard.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
+---
+eip:
+title: Subscription Token Standard for NFTs and Multi Tokens
+description: A standard interface for subscription tokens that gives holders subscriptions to NFTs and multi tokens
+author: Jules Lai (@julesl23)
+discussions-to: https://ethereum-magicians.org/t/eip-subscription-token-standard/8531
+status:
```
Comment (MicahZoltu) [inline EIPS/EIP-Subscription_Token_Standard.md:7]:
```suggestion
status: Draft
```

File: EIPS/EIP-Subscription_Token_Standard.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
+---
+eip:
```
Comment (MicahZoltu) [inline EIPS/EIP-Subscription_Token_Standard.md:2]:
```suggestion
eip: 4885
```

File: EIPS/eip-4393.md [hunk 3]
```diff
@@ -310,6 +310,8 @@ deposits can remain where they are when a user tips, and only transferred out to
 
 This standard does not specify the type of ERC-20 compatible deposits allowed. Indeed, could be tip tokens themselves. But it is RECOMMENDED that balances of the deposits be checked after transfer to find out the exact amount deposited to keep internal accounting consistent. In case, for example, the ERC-20 contract takes fees and hence reduces the actual amount deposited.
 
+This standard does not specify any functionality for refunds for deposits nor for tip tokens sent, it is left to the implementer to add this to their smart contract(s). The reasoning for this is to keep the interface light and not to enforce upon implementers the need for refunds but to leave that as a choice.
+
```
Comment (MicahZoltu) [inline EIPS/eip-4393.md:314]: Changes to existing EIPs should be done in their own PR, not as part of a new EIP PR.
```suggestion
```

**Comment by julesl23** [comment]
@MicahZoltu Thanks. I've made suggested changes. Let me know if there are more.
Cheers.

**Comment by MicahZoltu** [comment]
You'll have to wait for an editor to review the EIP, and I don't handle ERCs so it'll be someone else.  In the meantime, I recommend getting Travis CI to pass.  It is currently failing on a spelling mistake:
```
./EIPS/eip-4885.md:22: homogenous ==> homogeneous
```

File: EIPS/eip-4393.md [hunk 4]
```diff
@@ -310,6 +310,8 @@ deposits can remain where they are when a user tips, and only transferred out to
 
 This standard does not specify the type of ERC-20 compatible deposits allowed. Indeed, could be tip tokens themselves. But it is RECOMMENDED that balances of the deposits be checked after transfer to find out the exact amount deposited to keep internal accounting consistent. In case, for example, the ERC-20 contract takes fees and hence reduces the actual amount deposited.
 
+This standard does not specify any functionality for refunds for deposits nor for tip tokens sent, it is left to the implementor to add this to their smart contract(s). The reasoning for this is to keep the interface light and not to enforce upon implementors the need for refunds but to leave that as a choice.
+
```
Comment (MicahZoltu) [inline EIPS/eip-4393.md:314]: Please move this to a separate pull request.  Changes to an existing EIP should not be in the same PR as a new EIP submission.

Comment (julesl23) [reply EIPS/eip-4393.md]: Done

File: EIPS/eip-4885.md [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+
+When the subscription token balance falls to zero for a subscriber (signifying that the subscription has expired) then it is up to the implementer on how to handle this for their particular use case. For example, a provider may stop streaming media service to a subscriber. For an NFT that represents an image stored off-chain, perhaps the NFT's `uri` function no longer returns back a link to its metadata.
+
+### Caveats
+
+With some traditional subscription models based on fiat currencies, the subscribers' saved payment credentials are used to automatically purchase to extend the subscription period, at or just before expiry. This feature is not possible in this proposal specification as recurring payments will have to have allowance approved for signed by a subscriber for each payment when using purely cryptocurrencies.
+
+This proposal does not deal with pausing subscriptions directly, implementers can write their own or inherit off 3rd party smart contract abstractions such as OpenZeppelin's Pausable. In that case, `balanceOf` method would need extra logic and storage to account for the length of time the subscription tokens were paused.
+
+## Rational
```
Comment (lightclient) [inline EIPS/eip-4885.md:173]:
```suggestion
## Rationale
```

**Review by lightclient — CHANGES_REQUESTED** [review]

**Review by lightclient — APPROVED** [review]
